### PR TITLE
HDDS-5704. Ozone URI syntax description in help content needs to mention about ozone service id

### PR DESCRIPTION
**Pull request by Galsza**
## What changes were proposed in this pull request?

Updated the URI syntax description so that it contains examples of serviceId and port usage.

## What is the link to the Apache JIRA

[Improve the description of Ozone URI in Ozone Shell help message](https://issues.apache.org/jira/browse/HDDS-5704)

## How was this patch tested?

Running on a local docker cluster I've reproduced the message and also experimented on separate ways how the URIs can be used.

### CODE REVIEW

**Review by aswinshakil — APPROVED** [review]
LGTM +1

**Review by adoroszlai — COMMENTED** [review]
Thanks @Galsza for working on this.  I think we should distinguish between OM host name and OM service ID a bit more (and explain what the latter means, e.g. by referring to the config property that defines the service ID).

File: hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/Shell.java [hunk 1]
```diff
@@ -29,11 +29,16 @@
...
-      + "If they are not specified it will be identified from "
-      + "the config files.";
+  public static final String OZONE_URI_DESCRIPTION = "Ozone URI could either " +
+      "be a full URI or short URI.\n" +
+      "Full URI should start with o3://serviceId/ " +
+      "and can optionally also contain " +
+      "the port number: o3://serviceId:9999/\n" +
+      "Example of a full URI for a key:\no3://om/vol1/bucket1/key1\n" +
+      "With port number for a volume:\no3://om:9862/vol1/\n" +
+      "Not specified information will be identified from the config files.\n" +
```
[the remark this answers is not in the document]
Comment (Galsza) [reply hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/Shell.java]: Thank you for the review, fixed both issues.

**Review by DaveTeng0 — COMMENTED** [review]
LGTM~

**Comment by kerneltime** [comment]
@adoroszlai should we move this to be `ofs` as `o3` should most likely get deprecated?

**Comment by adoroszlai** [comment]
> should we move this to be `ofs` as `o3` should most likely get deprecated?

This is for the `o3://` URLs for OM client, not related to OFS/O3FS.

**Review by kerneltime — APPROVED** [review]

**Review by adoroszlai — APPROVED** [review]

**Pull request merged**